### ISSUE TITLE
Screen sharing crashes when no sources are selected

### ISSUE THREAD

**Issue by BlueEyedCrow** [smartfrigde/armcord#762]
**Describe the bug**
Screen sharing crashes when no sources are selected. 

**To Reproduce**
1. Click on "Share your Screen" 
2. On the source portal click on "x" to close the window

**Expected behavior**
Expecting to be able to cancel the sharing without the silent crash.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux 6.11.3-arch1-1 EndeavourOS x86_64
 - Method of installation: AppImage
 - Version Legcord-1.0.2-linux-x86_64

**Additional context**
Core Dump:
https://pastebin.com/NrpeNKfn

**Comment by SomeAspy** [comment]
I've known about this for a while, and could have sworn there was an open issue relating to it.
I've attempted to fix it several times with no luck. Although I'm unsure how you are only now just experiencing this, as this has been a long standing issue (related to upstream electron if I remember correctly? again, I could have sworn I documented this prior) 

edit:
Ah, I think I've found it. It seems to have progressed from an annoying but to a crash. cool. https://github.com/Legcord/Legcord/issues/594
